### PR TITLE
BaSP-TG-83: Super Admin redux - Create, update and delete

### PR DESCRIPTION
**Pull request by matescarabino**
- Create constants to create, update and delete superAdmins
- Create actions to create, update and delete superAdmins
- Create thunk to create, update and delete superAdmins
- Complete superAdmins reducer to create, update and delete
- Connect list component with store to get and delete superAdmins
- Conectect form with store to update and create superAdmins

For QAs:
See "how to test?" on Trello:
[TG-83](https://trello.com/c/mQU0raqC)

### CODE REVIEW

File: src/Components/SuperAdmins/Form/Form.module.css [hunk 1]
```diff
@@ -5,6 +5,20 @@
...
+    justify-content: center;
+    width: 100%;
+    font-size: 20px;
+    margin-left: 10%;
+    margin-right: 10%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-weight: bold;
+    text-transform: uppercase;
+    padding: 5px;
```
Comment (CristianBucci) [inline src/Components/SuperAdmins/Form/Form.module.css:19]: Padding should be with margin rules!

File: src/Components/SuperAdmins/Form/Form.module.css [hunk 2]
```diff
@@ -5,6 +5,20 @@
     color: #000000;
 }
 
+.cardTitle {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    font-size: 20px;
```
Comment (CristianBucci) [inline src/Components/SuperAdmins/Form/Form.module.css:12]: Font rules should be under color rules!

File: src/Components/SuperAdmins/Form/Form.js [hunk 3]
```diff
@@ -6,147 +6,99 @@ import Input from '../../Shared/Inputs';
...
-        lastName: response.data.lastName,
-        email: response.data.email,
-        password: response.data.password
-      });
-    } catch (error) {
-      setModalContent({ title: 'ERROR!', content: `Could not GET SuperAdmin! ${error.message}` });
-      setShowModalMessage(true);
-    }
+  const modalFunction = () => {
+    modalContent.title.includes('SUCCESS') ? redirect() : null;
```
Comment (MaxiG-dev) [inline src/Components/SuperAdmins/Form/Form.js:72]: You can replace 
`modalContent.title.includes('SUCCESS') ? redirect() : null;`
with
`modalContent.title.includes('SUCCESS') && redirect();`

File: src/Components/SuperAdmins/index.js [hunk 4]
```diff
@@ -1,54 +1,72 @@
...
 
-  useEffect(() => {
-    dispatch(getSuperAdmin());
-  }, []);
+  const redirect = () => {
+    props.history.push('/super-admins');
+  };
+
+  const modalFunction = () => {
+    modalContent.title.includes('SUCCESS') ? redirect() : null;
```
Comment (MaxiG-dev) [inline src/Components/SuperAdmins/index.js:46]: You can replace 
`modalContent.title.includes('SUCCESS') ? redirect() : null;`
with
`modalContent.title.includes('SUCCESS') && redirect();`

Comment (matescarabino) [reply src/Components/SuperAdmins/index.js:46]: Done

File: src/Components/SuperAdmins/Form/Form.js (hunk 3 above)
Comment (matescarabino) [reply src/Components/SuperAdmins/Form/Form.js:72]: Done

**Review by ignacioaristo — APPROVED** [review]

File: src/Components/SuperAdmins/Form/Form.module.css (hunk 1 above)
Comment (matescarabino) [reply src/Components/SuperAdmins/Form/Form.module.css:19]: Done

File: src/Components/SuperAdmins/Form/Form.module.css (hunk 2 above)
Comment (matescarabino) [reply src/Components/SuperAdmins/Form/Form.module.css]: Done

**Review by CristianBucci — APPROVED** [review]
Everything looks great!

**Review by NicoSan02 — APPROVED** [review]

**Pull request merged**